src/app/constants/member-dialog-data-config.constant.ts

```diff
@@ -0,0 +1,173 @@
+import { MemberDialogItem } from '../interfaces/member-dialog.item.interface';
+
+export const MEMBER_DIALOG_DATA: readonly MemberDialogItem[] = [
+  {
+    name: 'Héja Benedek',
+    memberId: 'heja-benedek',
+    description:
+      'Zenészcsaládba születtem. Édesapám, Héja László (aki nem mellesleg zenekarunk állandó vendégkarmestere) az Operaház zenekarának volt hosszú évtizedekig tagja, édesanyám, Fáth Helga pedig hegedűtanárként dolgozott a III. kerületi Aelia Sabina Zeneiskolában. Bátyám, Héja Domonkos, karmester, az Óbudai Danubia Zenekar alapítója. Így nem meglepő, hogy én is ezt a pályát választottam. A fentebb említett zeneiskola diákjaként korán megismerkedtem a kamarazenélés, illetve a zenekari játék örömével. Zenetanárként is ide tértem vissza, ahol az intézmény vonószenekarát vezetem közel másfél évtizede. Itteni tanítványaim - élükön Nánay Mihály barátommal - kértek meg, hogy segítsek elindítani egy új együttest, amely az iskola már végzett diákjaiból állt össze. Azóta is együtt vagyunk, a társaság pedig hamar kibővült más, szintén zenekedvelő tagokkal. Túlzás nélkül állíthatom: olyan ez, mint egy második család. A szó legszebb értelmében.',

```

Comment:
ezeket is forditasi kulcsba kene kiszervezni, es itt a leiras csak a forditasi kulcsot tartalmazna tehat `descriptionTranslationKey`, nev eseten is el lehet gondolni, angolra valtas eseten a nev felcserelodik